dftimewolf/lib/state.py

```diff
@@ -238,6 +239,10 @@ def StoreContainer(self, container: "interface.AttributeContainer") -> None:
       container (AttributeContainer): data to store.
     """
     with self._state_lock:
+      calling_class = inspect.stack()[1][0].f_locals["self"].__class__.__name__
+      container.src_module = calling_class
```

Comment:
```suggestion
      container.src_module_class = calling_class
```

Reply:
Based on the above, this is now `src_module_name`